modules/statics/src/account.ts

```diff
@@ -254,6 +258,22 @@ export class SolCoin extends AccountCoinToken {
   }
 }
 
+/**
+ * The AVAX network support tokens
+ * AVAX Tokens work similar to ERC20 coin, but need
+ * to asset dXChain to the CrossChain
+ */
+export class ARC20Token extends ContractAddressDefinedToken {
+  public assetIdXChain: string;
+  constructor(options: ARC20CoinContructorOptions) {
+    super({
+      ...options,
+    });
+
+    this.assetIdXChain = options.assetIdXChain;
+  }
+}
```

Comment:
Should we support regular ERC20 token in avax?
Should ARC20 token extends ERC20 ?
if both answers are yes, i suggest this change.

```suggestion
/**
 * The AVAX C Chain network support tokens
 * AVAX C Chain Tokens are ERC20 coin
 */
export class AvaxERC20Token extends ContractAddressDefinedToken {
  constructor(options: ERC20CoinContructorOptions) {
    super({
      ...options,
    });
}
}

/**
 * The AVAX C chain network support tokens
 * AVAX Native Tokens work similar to ERC20 coin, but need
 * to asset dXChain to the CrossChain
 */
export class ARC20Token extends AvaxERC20Token {
  public assetIdXChain: string;
  constructor(options: ARC20CoinContructorOptions) {
    super({
      ...options,
    });

    this.assetIdXChain = options.assetIdXChain;
  }
}
```

Reply:
Done!